trinity/components/eth2/eth1_monitor/exceptions.py

```diff
@@ -0,0 +1,19 @@
+from eth_utils import ValidationError
+
+
+class Eth1MonitorError(Exception):
+    pass
```

Comment:
nitpick: now Trinity prefers to use `...` for declaring exceptions.
```suggestion
    ...
```